src/lua/compat.lua

```diff
@@ -15,14 +14,32 @@ local options_format = {
     action         = 'function/nil',
 }
 
+local FIBER_CHANNEL_GRACEFUL_CLOSE_BRIEF = [[
+Whether fiber channel should discard all messages in its buffer after being
+closed. New behavior allows user to get all messages that were put into buffer
+before close afterwards, by marking channel as read-only. The read-only channel
```

Comment:
> before close afterwards

Oh my brain... Can you explain it like I'm five, please?

Reply:
Done.